authentication-demo-service/pom.xml

```diff
@@ -221,22 +202,19 @@
 		<dependency>
 			<groupId>commons-codec</groupId>
 		  	<artifactId>commons-codec</artifactId>
-		  	<version>1.11</version>
 		</dependency>
 		<dependency>
 			<groupId>io.mosip.kernel</groupId>
 			<artifactId>kernel-templatemanager-velocity</artifactId>
-			<version>1.2.0.1-SNAPSHOT</version>
+			<version>1.2.1-java21-SNAPSHOT</version>

```

Comment:
If it is develop branch then version should be 1.2.1-SNAPSHOT

Reply:
Done.